projects/aca-content/src/lib/components/search/search-input/search-input.component.html

```diff
@@ -11,24 +11,25 @@
     <input
       matInput
       [attr.aria-label]="'SEARCH.INPUT.ARIA-LABEL' | translate"
+      class="app-input-form-field-input"
       [type]="'text'"
       [disabled]="true"
       [value]="searchedWord"
       [placeholder]="'SEARCH.INPUT.PLACEHOLDER' | translate"
     />
 
     <div matSuffix class="app-suffix-search-icon-wrapper">
-      <mat-icon>arrow_drop_down</mat-icon>
+      <mat-icon class="app-drop-down-icon">arrow_drop_down</mat-icon>
     </div>
 
-    <button mat-icon-button matSuffix class="app-suffix-search-icon-wrapper app-close-icon" (click)="exitSearch()">
-      <mat-icon>close</mat-icon>
+    <button mat-icon-button matSuffix class="app-suffix-search-icon-wrapper app-close-button" (click)="exitSearch()">
+      <mat-icon class="app-close-icon">close</mat-icon>
     </button>
   </mat-form-field>
 </div>
 
 <mat-menu #searchOptionsMenu="matMenu" [overlapTrigger]="true" class="app-search-options-menu">
-  <div (keydown.tab)="$event.stopPropagation()" (keydown.shift.tab)="$event.stopPropagation()" tabindex=0>
+  <div (keydown.tab)="$event.stopPropagation()" (keydown.shift.tab)="$event.stopPropagation()" tabindex=0 class="test">
```

Comment:
class="test" - should be removed probably?

Reply:
right, ty